radiation/python/radsw/radsw_param.py

```diff
@@ -0,0 +1,272 @@
+import numpy as np
+
+# Parameter constants for SW band structures
+
+# band range lower index
+nblow = 16
+# band range upper index
```

Comment:
do you know the reason for a lower and upper index?

Reply:
I'm not totally sure. It seems the SW bands are numbered 16-29, and in the fortran they assign values to those indices in the "nbands" dimension. I just made it indices 0-13 instead, for simplicity.